lib/Maintenance/ExecutorInterface.php

```diff
@@ -20,14 +20,16 @@
  */
 interface ExecutorInterface
 {
+    public function executeTask(string $name, bool $force = false);
+
     /**
      * Execute the Maintenance Task
      *
      * @param array $validJobs
      * @param array $excludedJobs
      * @param bool $force
      */
-    public function executeMaintenance(array $validJobs = [], array $excludedJobs = [], bool $force = false);
+    public function executeMaintenance(array $validJobs = [], array $excludedJobs = [], bool $force = false, bool $async = false);
```

Comment:
```suggestion
    public function executeMaintenance(array $validJobs = [], array $excludedJobs = [], bool $force = false);
```
We don't need the `$async` status here, as we're always adding the tasks to the queue.